src/app/hearing-part/components/hearing-search-table/hearing-search-table.component.html

```diff
@@ -11,52 +11,55 @@
   </ng-container>
 
   <ng-container matColumnDef="caseNumber">
-    <mat-header-cell *matHeaderCellDef > Case Number </mat-header-cell>
-    <mat-cell *matCellDef="let element"> {{element.caseNumber}} </mat-cell>
+    <mat-header-cell *matHeaderCellDef> Case Number</mat-header-cell>
+    <mat-cell *matCellDef="let element">
+      <a id="view-hearing-{{element.id}}" routerLink="/home/hearing/{{element.id}}">{{element.caseNumber}}</a>
```

Comment:
if I'd be really picky i'd say the path for routerLink should be generated in a method in component so that it's easier to unit tests. For me It's fine as is though